README.rst

```diff
@@ -247,6 +247,10 @@ Choose from the list of available rules:
     whose lines all start with an asterisk (``phpdocs_like``) or any
     multi-line comment (``all_multiline``); defaults to ``'phpdocs_only'``
 
+* **array_indentation**
+
+  Multiline arrays must be indented with configured indentation.
```

Comment:
actually, I would drop ` with configured indentation` part, it's misleading, we don't have config for this fixer.

you meant global config, but that is used across many rules, and we don't mention that "global config" for them.

Reply:
Updated.